content/whats-next/whats-next-mid-term.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "title": "Product identifier generator",
+    "description": "We want the PIM to be responsible for creating unique product identifiers on the fly when importing/creating new products, based on nomenclatures.",
+    "areas": ["Automation","Governance"]
+  },
+  {
+    "title": "Several product identifiers for one product",
+    "description": "We want to allow the use of several identifiers because in most of the cases, both GTIN and SKU are needed. Some of you even use, in addition to those two, an internal identifier.",
```

Comment:
```suggestion
    "description": "We want to allow the use of several identifiers because in most cases, both GTIN and SKU are needed. Some of you even use an internal identifier in addition to those two.",
```